Python/omevv/v1/scripts/GetOMIVVProfileDetails.py

```diff
@@ -1,30 +1,48 @@
 import argparse
-import sys
-import csv
 import time
-import warnings
-from webbrowser import get
-from omivv_models import consoleDecoder,repoProfileDecoder
+from omivv_models import consoleDecoder, repoProfileDecoder, ClusterProfileDecoder
 from utilities import Utilities
+import requests, json, base64
 
-import requests,json,base64
 
-class RepoDetails:
-    def __init__(self,omivv_ip,omivv_username,omivv_pwd,domain):
+class ProfileDetails:
+    def __init__(self, omivv_ip, omivv_username, omivv_pwd, domain):
         self.omivv_username = omivv_username.strip()
         self.headers = {'content-type': 'application/json'}
         self.domain = domain
         self.omivv_password = omivv_pwd.strip()
-        self.login_url = "https://%s/Spectre/api/rest/v1/Services/AuthenticationService/login"%(omivv_ip)
-        self.console_url = "https://%s/Spectre/api/rest/v1/Services/ConsoleService/Consoles"%(omivv_ip)
-        self.context_url = "https://%s/Spectre/api/rest/v1/Services/ConsoleService/OperationalContext"%(omivv_ip)
-        self.repo_url = "https://%s/Spectre/api/rest/v1/Services/PluginProfileService/RepositoryProfiles"%(omivv_ip)
+        self.login_url = "https://%s/Spectre/api/rest/v1/Services/AuthenticationService/login" % (omivv_ip)
+        self.logoff_url = "https://%s/Spectre/api/rest/v1/Services/AuthenticationService/logoff" % omivv_ip
+        self.console_url = "https://%s/Spectre/api/rest/v1/Services/ConsoleService/Consoles" % (omivv_ip)
+        self.context_url = "https://%s/Spectre/api/rest/v1/Services/ConsoleService/OperationalContext" % (omivv_ip)
+        self.repo_url = "https://%s/Spectre/api/rest/v1/Services/PluginProfileService/RepositoryProfiles" % (omivv_ip)
+        self.baseline_url = "https://%s/Spectre/api/rest/v1/Services/PluginProfileService/ClusterProfiles" % omivv_ip
         self.omivv_encoded_cred = self.encode_cred(self.omivv_username, self.omivv_password)
-        self.payload_cred_dict= {}
+        self.payload_cred_dict = {}
         self.payload_cred_dict['console'] = 'consoleUserCredential'
         self.payload_cred_dict['api'] = 'apiUserCredential'
         self.retry = 3
-
+        # Generating the bearer token to be used in this script
+        self.headers['Authorization'] = self.omivv_encoded_cred
```

Comment:
Have this login as part of function, for more readability

Reply:
@pavithra-mahadev Done